include/tesla.hpp

```diff
@@ -1059,7 +1061,33 @@ namespace tsl {
                 static PlFontData stdFontData, extFontData;
 
                 // Nintendo's default font
-                TSL_R_TRY(plGetSharedFontByType(&stdFontData, PlSharedFontType_Standard));
+                u64 languageCode;
+                if (R_SUCCEEDED(setGetSystemLanguage(&languageCode)))
+                {
```

Comment:
Please follow the style used elsewhere in libtesla too. `{` goes on the same line as the control flow statement